docker/gcp_pudl_etl.sh

```diff
@@ -337,16 +338,16 @@ gcloud storage --quiet cp "$LOGFILE" "$PUDL_GCS_OUTPUT"
 rm -f ~/.aws/credentials
 
 # Notify slack about entire pipeline's success or failure;
-if [[ $ETL_SUCCESS == 0 && \
-      $SAVE_OUTPUTS_SUCCESS == 0 && \
-      $UPDATE_NIGHTLY_SUCCESS == 0 && \
-      $UPDATE_STABLE_SUCCESS == 0 && \
-      $DATASETTE_SUCCESS == 0 && \
-      $CLEAN_UP_OUTPUTS_SUCCESS == 0 && \
-      $DISTRIBUTION_BUCKET_SUCCESS == 0 && \
-      $GCS_TEMPORARY_HOLD_SUCCESS == 0 && \
-      $ZENODO_SUCCESS == 0
-]]; then
+if [[ $ETL_SUCCESS == 0 &&
+    $SAVE_OUTPUTS_SUCCESS == 0 &&
+    $UPDATE_NIGHTLY_SUCCESS == 0 &&
+    $UPDATE_STABLE_SUCCESS == 0 &&
+
```

Comment:
Is this awkward empty line + the post-conditional comment so that we remember to put datasette back in? If so I think we should probably explain the intent of this somewhat strange looking block with an expanded comment in line 340.